backends/ebpf/psa/ebpfPsaTable.cpp

```diff
@@ -690,7 +690,6 @@ void EBPFTablePSA::emitKeysAndValues(CodeBuilder *builder,
             CHECK_NULL(fieldName);
             builder->emitIndent();
             builder->appendFormat("%s.%s = ", keyName.c_str(), fieldName.c_str());
-            auto mtdecl = program->refMap->getDeclaration(keyElement->matchType->path, true);
```

Comment:
This seems unrelated?

Reply:
Yes, it is a separate commit. Another unused variable.